migrations/sql/V1.21__Alter_Mine_Location_Add_Geoms.sql

```diff
@@ -0,0 +1,5 @@
+-- SRID=3005 is ‘BC Albers’ projection, chosen as the standard projection for this application.
+ALTER TABLE mine_location ADD COLUMN geom GEOMETRY(POINT, 3005);
+UPDATE mine_location SET geom = ST_SetSRID(ST_MakePoint(longitude, latitude),3005) WHERE longitude IS NOT NULL AND latitude IS NOT NULL;
```

Comment:
Where is the number 3005 coming from?
If i'm not mistaken all our maps use `4326`. Although im not an expert in these 😅

Reply:
We will move forward and create another ticket to discuss the projection we use.